sasjslint-schema.json

```diff
@@ -8,14 +8,20 @@
     "noTrailingSpaces": true,
     "noEncodedPasswords": true,
     "hasDoxygenHeader": true,
-    "noSpacesInFileNames": true
+    "noSpacesInFileNames": true,
+    "lowerCaseFileNames": true,
+    "maxLineLength": 80,
+    "noTabIndentation": true
```

Comment:
it would be great if we can change this to `indentationMultiple`:<integer>

This would represent a check to ensure every line was indented by say, 2 or 3 or 4 _spaces_.  Zero, null or undefined would be 'off'.

So it would simultaneously eliminate tabs as well as ensure consistent indentation.

Indeed 4 spaces is a multiple of 2, but given 99% of indentation issues are relating to the use of tabs or 3 spaces, I think this is ok.  Figuring out which lines should follow strict indentation rules is a very. big. job.

Reply:
@allanbowe This has now been implemented - `indentationMultiple` is a configurable number value independent of the `noTabIndentation` flag.
When the value for `indentationMultiple` is set to zero, we will ignore any indentation violations.
When it has not been set (i.e. property not present in the `.sasjslint` file), we will assume 2 as the default.